components/blocks/autofunction.module.css

```diff
@@ -16,6 +16,7 @@
 
 .Form {
   @apply relative my-4 md:my-0 ml-0 md:ml-auto;
+  max-width: fit-content;
```

Comment:
@snehankekre We can use `max-w-fit` here as the Tailwind equivalent, see [here](https://tailwindcss.com/docs/max-width)

Reply:
Fixed on 6a5bc20